docs/source/configuration.rst

```diff
@@ -145,57 +145,75 @@ You can also add directed assembly/C test in the testlist
-| enable_ebreak_in_debug_rom  | Generate ebreak instructions inside debug ROM     | 0       |
-+-----------------------------+---------------------------------------------------+---------+
-| set_dcsr_ebreak             | Randomly enable dcsr.ebreak(m/s/u)                | 0       |
-+-----------------------------+---------------------------------------------------+---------+
-| randomize_csr               | Fully randomize main CSRs (xSTATUS, xIE)          | 0       |
-+-----------------------------+---------------------------------------------------+---------+
++---------------------------------+---------------------------------------------------+---------+
+| Option                          | Description                                       | Default |
++=================================+===================================================+=========+
+| num_of_tests                    | Number of assembly tests to be generated          | 1       |
++---------------------------------+---------------------------------------------------+---------+
+| num_of_sub_program              | Number of sub-program in one test                 | 5       |
++---------------------------------+---------------------------------------------------+---------+
+| instr_cnt                       | Instruction count per test                        | 200     |
++---------------------------------+---------------------------------------------------+---------+
+| enable_page_table_exception     | Enable page table exception                       | 0       |
++---------------------------------+---------------------------------------------------+---------+
+| enable_unaligned_load_store     | Enable unaligned memory operations                | 0       |
++---------------------------------+---------------------------------------------------+---------+
+| no_ebreak                       | Disable ebreak instruction                        | 1       |
++---------------------------------+---------------------------------------------------+---------+
+| no_wfi                          | Disable WFI instruction                           | 1       |
++---------------------------------+---------------------------------------------------+---------+
+| set_mstatus_tw                  | Enable WFI to be treated as illegal instruction   | 0       |
++---------------------------------+---------------------------------------------------+---------+
+| no_dret                         | Disable dret instruction                          | 1       |
++---------------------------------+---------------------------------------------------+---------+
+| no_branch_jump                  | Disable branch/jump instruction                   | 0       |
++---------------------------------+---------------------------------------------------+---------+
+| no_load_store                   | Disable load/store instruction                    | 0       |
```

Comment:
should this be removed since it's not used?

Reply:
Good point, I forgot to get rid of it.